LBH.AdultSocialCare.Api/V1/Controllers/ResidentialCare/ResidentialCarePackageController.cs

```diff
@@ -74,5 +82,52 @@ public async Task<ActionResult<ResidentialCarePackageResponse>> Get(Guid residen
             }
         }
 
+        /// <summary>Change the residential care package status.</summary>
+        /// <param name="residentialCarePackageId"></param>
+        /// <param name="statusId"></param>
+        /// <returns>The residential care package response model.</returns>
+        [ProducesResponseType(typeof(ResidentialCarePackageResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
+        [HttpPut("changeStatus")]
+        public async Task<ActionResult<NursingCarePackageResponse>> ChangeStatus(
+            Guid residentialCarePackageId, int statusId)
```

Comment:
Another change status action used an input model, I think it makes sense to use parameters for such a small action, but we should be consistent across these change status actions